.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Test
         run: .circleci/test.sh
-        timeout-minutes: 25
+        timeout-minutes: 30
```

Comment:
Might be time to look into running tests in parallel to claw back our turnaround time

Reply:
GitHub Actions doesn't seem to have a native option. Might be able to do something with https://github.com/hugeinc/behave-parallel ?